src/js/edu-benefits/5490/config/form.js

```diff
@@ -201,6 +198,9 @@ const formConfig = {
               previousBenefits: _.merge(
                 _.unset('properties.veteranFullName', previousBenefits),
                 {
+                  // Doesn't let us pass even if view:claimedSponsorService is false
+                  //  (which makes the ssn box is hidden)
+                  // required: ['veteranSocialSecurityNumber'],
```

Comment:
Oh, yeah this works now (just confirmed again), I just forgot to remove the comment after I found `'ui:required'`.